adapters/brightroll/brightroll.go

```diff
@@ -189,8 +228,18 @@ func getMediaTypeForImp(impId string, imps []openrtb.Imp) openrtb_ext.BidType {
 	return mediaType
 }
 
-func NewBrightrollBidder(endpoint string) *BrightrollAdapter {
-	return &BrightrollAdapter{
-		URI: endpoint,
+func NewBrightrollBidder(endpoint string, extraAdapterInfo string) *BrightrollAdapter {
+
+	var extraInfo ExtraInfo
```

Comment:
Please consider adding unit tests for this function

Reply:
Yes, there are testcases (existing testcases) updated to cover this function

Reply:
This is what the test coverage for `NewBrightrollBidder` looks like. @mansinahar, what were you looking for coverage on?

<img width="710" alt="Screen Shot 2020-09-03 at 1 49 52 PM" src="https://user-images.githubusercontent.com/1653437/92149724-645b6f00-edec-11ea-8f00-49477f77c447.png">

Reply:
I believe the existing tests i.e. the `RunJSONBidderTest` are more integration tests for the entire adapter code. I was more suggesting to add unit tests for this particular `NewBrightrollBidder` method as this is no longer a simple constructor that takes parameters and assigns them to the object. I'd expect this to be a simple test consisting of two test cases, one that calls the `NewBrightrollBidder` function with an empty string for `extraAdapterInfo` and another that calls it with a non-empty, valid `extraAdapterInfo` string.

Reply:
@mansinahar Can you please provide some guidance on where/how to add such testcases in the existing adapter framework or please point me to any such existing testcases to refer.